HappyAnding/HappyAnding/Views/Components/AdminCurationCell.swift

```diff
@@ -7,14 +7,74 @@
 
 import SwiftUI
 
+/*
+ 어드민 큐레이션 썸네일입니다.
+ 
+ #parameters
+ - curationThumbnail: 썸네일 배경의 이미지명
+ - title: 큐레이션 제목
+ - subtitle: 큐레이션 부제목
+ 
+ #description
+ - 어드민 큐레이션 데이터가 만들어진 후 데이터 연결이 필요합니다.
+ - padding 값이 설정되어있지 않습니다. 사용 시 padding 설정 후 사용해주세요.
+ */
+
 struct AdminCurationCell: View {
+    
+    let curationThumbnail: String
+    let title: String
+    let subtitle: String
+    
+    let cornerRadius: CGFloat = 12
+    
     var body: some View {
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
+        ZStack(alignment: .top) {
+            NavigationLink(destination: ReadCurationView()) {
+                EmptyView()
+            }
+            titleAndSubtitle
+                .padding([.leading, .bottom, .trailing], 24.0)
+                .padding(.top, 184)
+                .background(backgroundImage)
+        }
+    }
```

Comment:
```suggestion
        }
        .padding(16)
     }
```

해당 뷰를 호출할 때마다 padding이 변하는 것이 아니라면, 재사용 셀에서 적용해주는게 좋을 것 같아요

Reply:
이 컴포넌트가 safearea를 무시하고 횡스크롤되는 뷰에 사용되어서 횡스크롤 구현 시 각 컴포넌트 사이에 padding을 주는 게 효율적이라고 생각했습니다! 어떤 방향이 더 적합할까요?

Reply:
컴포넌트마다 padding 값을 다르게 주어야한다면 그대로 유지해주세요!
그런게 아니라면 재사용 셀 안에서 해결하는게 좋을 것 같아요~

Reply:
컴포넌트마다 다르게 적용될 것 같지는 않습니다 그러면 우측에만 padding값 8 주고 머지하겠습니다!